pydevd.py

```diff
@@ -256,6 +258,9 @@ def __init__(self):
         self.show_return_values = False
         self.remove_return_values_flag = False
 
+        # this dict is used for frame evaluation, it holds thread_id -> set of frames
+        self.disable_tracing_after_exit_frames = {}
```

Comment:
Yes, but we still need a flag for disabling frame evaluation (if, for example, breakpoint was added during running). So, I've added a field "do_not_use_frame_eval"